kvdb-rocksdb/Cargo.toml

```diff
@@ -31,3 +31,7 @@ ethereum-types = { path = "../ethereum-types" }
 kvdb-shared-tests = { path = "../kvdb-shared-tests", version = "0.2" }
 rand = "0.7.2"
 tempdir = "0.3.7"
+keccak-hash = { path = "../keccak-hash" }
+sysinfo = "0.11.7"
+ctrlc = "3.1.4"
+time = "0.2.9"
```

Comment:
I think the API changed and now CI's hating on us. :/

Reply:
Alternatively, we could use https://docs.rs/humantime/2.0.0/humantime/fn.format_rfc3339.html

Reply:
I have no idea why the made 0.2 version completely unusable, returned 0.1